pom.xml

```diff
@@ -271,8 +271,8 @@
         </dependency>
         <dependency>
             <groupId>org.mockito</groupId>
-            <artifactId>mockito-all</artifactId>
-            <version>1.10.19</version>
+            <artifactId>mockito-core</artifactId>
+            <version>3.11.0</version>
```

Comment:
I needed to use a more recent version of mockito (mockito-all is currently deprecated). Otherwise, the interaction with the `org.hamcrest.Matchers` comparators fails